DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/CourseDelegates/Index.cshtml

```diff
@@ -71,7 +71,7 @@
           </form>
         </div>
 
-        <partial name="SearchablePage/Configurations/_ThreeQuarterWidthSortAndFilter" model="Model.CourseDetails" />
+        <partial name="SearchablePage/Configurations/_FullWidthSearchSortAndFilter" model="Model.CourseDetails" />

```

Comment:
Not a good place for this comment, but I noticed that the links to Edit Completed Date etc. for Course Delegate cards are removed when the progress record is removed, but the Edit Admin Fields links are still present. We could do with removing them in this case to make them consistent. Might need to add a note about testing this to the ticket as it technically isn't part of the scope.

Reply:
Nevermind this, these are all going to be tackled in the Delegate Progress page merging refactor. I'll resolve this.